Contributing.md

```diff
@@ -30,10 +30,19 @@ This project uses clang-format (stable branch) to check for common style issues.
 
 In most cases, clang-format can and **should** be used to automatically reformat code and solve most formatting issues.
 
-- To run clang-format on all staged files:
+It can be installed using your distro package manager (on Linux) or from [LLVM binary packages](http://releases.llvm.org/download.html) for the latest version.
+
+- To run clang-format on all staged files, use the following command:
   ```
-  git diff --cached --name-only | egrep '[.](cpp|h|mm)$' | xargs clang-format -i
+  git diff --cached --name-only | egrep '[.](cpp|h|mm)$' | xargs clang-format -style=file -i
   ```
+  
+  - If you can't execute clang-format on Windows, try locating `clang-format.exe` in the LLVM installation and place it into the `...Git\usr\bin` directory.
+  
+- VS2017 users can install the [clang-format extension](https://marketplace.visualstudio.com/items?itemName=HansWennborg.ClangFormat#overview).
+
+   - Preferrably run the VSIX installer while Visual Studio is not running.
```

Comment:
Is stopping VS really necessary?

Reply:
You have to stop VS, otherwise it can't install. I don't think we should keep this sentence, though... It's not just "preferably", and besides, it's unnecessary for us to point this out because the installer already points it out.

Reply:
Ah okay, thanks for the clarification. I agree we should remove this sentence then.